CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.9)
+
+project(VulkanMemoryAllocator)
+
+set(CMAKE_DISABLE_IN_SOURCE_BUILD ON)
+
+find_package(Vulkan REQUIRED)
+
+# Put binaries into bin folder
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_SOURCE_DIR}/bin)
```

Comment:
Will this affect the build of my project?

Reply:
On Windows this makes Visual Studio output the binary into the `bin` folder. Usually the binary belongs to the `build` folder which is excluded from the git repository.

Reply:
Sure... but if my application uses VMA, and VMA's CMakeLists.txt sets the runtime output directory, will that affect what directory Visual Studio outputs my own application to? The point is that I don't want it to

Reply:
Oh that is a good point I didn't think of this! I only added it because the same directory also contains the compiled shaders. I will remove this setting and just change the working directory of the visual studio debugger. Since that's the only platform which supports VmaExample, that should do.

Reply:
Setting the output directory to `bin` is also bad because this means it's not in the gitignore.

Reply:
Yeah...

Reply:
Done.